DependencyInjection/Compiler/ProfilerControllerPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @copyright 2016 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\ContentfulBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\Config\FileLocator;
+
+class ProfilerControllerPass implements CompilerPassInterface
+{
+    /**
+     * You can modify the container here before it is dumped to PHP code.
```

Comment:
It'd be great if you could replace this with a short sentence what this specific method does. For example `Loads the definition for the ProfilerController when the profiler is present.`